src/constants.ts

```diff
@@ -3,5 +3,14 @@ export enum Categories {
   MANGA = "manga",
 }
 
+export enum ErrorResponseMessages {
+  SER01 = "Validation error",
+  SER02 = "Entry not found error",
+  SER03 = "Incorrect username or password",
+  SER04 = "Entry already exists",
+  SYS01 = "Unexpected error",
+  SYS02 = "Unexpected error",
+}
+
```

Comment:
Creo que es mejor tener estos como atributos en la clase que muestra tales mensajes. Así, el mensaje se puede personalizar más de acuerdo al contexto del error.

Aquí hay un ejemplo de eso:
<https://github.com/DiegoAvmor/uady-spa-project/blob/master/src/app/views/sign-in-view/sign-in-view.component.ts#L32>